assets/src/containers/GradeDistribution.js

```diff
@@ -37,14 +37,14 @@ function GradeDistribution (props) {
   const [gradeLoaded, gradeError, gradeData] = useGradeData(courseId)
   const [userSettingLoaded, userSetting] = useUserSetting(courseId, 'grade')
   const [settingChanged, setSettingChanged] = useState(false)
-  const [showGrade, setShowGrade] = useState(false)
+  const [showGrade, setShowGrade] = useState(true)
 
   useEffect(() => {
     if (userSettingLoaded) {
       if (isObjectEmpty(userSetting.default)) {
-        setShowGrade(false)
+        setShowGrade(true)
       } else {
-        setShowGrade(userSetting.default !== 'False')
+        setShowGrade(userSetting.default === 'False' ? false : true)
```

Comment:
logic changed to show My grades by default

Reply:
I think `setShowGrade(userSetting.default === 'False' ? false : true)` and `setShowGrade(userSetting.default === 'False' )` are logically equivalent.

Reply:
Changed it.